### ISSUE TITLE
Minor UI Issue: Horizontal Alignment of Version Info

### ISSUE THREAD

**Issue by rutskiuk** [mylar3/mylar3#1523]
The problem is quite simple
For a little while I've noticed that the horizontal alignment of the bottom lines of the comic list is shifted to the left.
It's as though the page browser and key has been shifted to the same line as the summary, line, version information and status pushing everything over to the left.
The problem isn't cache related as happens with every browser and on ever machine I have Mylar installed on.
The problem is limited to the comics list, every other page appears fine.
There isn't an error I can paste as such

![image](https://github.com/mylar3/mylar3/assets/25068977/bb049bb4-9ce0-48c5-830d-d1b7548cef70)

**Environment (please complete the following information):**
 - OS: Win10
 - Version: bbefdf9d8496e0ae8f395a771ac815fed16804b9 (python3-dev)

**Comment by evilhero** [comment]
Yeah I just realized what the problem is due to a similar issue being brought up (#1530).

If you set ``alpha_index = False`` in the config.ini, the spacing should be back to normal, but without the alphabetical categorization by letter. It will be much, much faster loading that particular page tho, but you'll have to use the filter box to search as opposed to clicking by specific letter.

I'll be looking into it later today, but unsure of a timeframe atm..

**Comment by rutskiuk** [comment]
Thank you for the suggestion, it does run much quicker however the offset is still there (tested incognito to be sure). 

Thanks for taking a look

**Comment by evilhero** [comment]
Try sorting by status on that page, and seeing if there is a series that is in a status of loading, removed or failed.

The alphaindex does play a factor, but I'm also believing that it's due to a series on your watchlist causing some display issues (ie. Series that jump to 2 display rows instead of one)

**Comment by rutskiuk** [comment]
Well it just gets curiouser, I can no longer sort by status, if I click to sort by status the continuing and ended are seemingly randomly scattered. Actually, not randomly at all, sorted by Title, alphabetically.
I cannot see any removed, loading or failed series, I changed the view to show all and tried a search through both the browser and the filter, none show anything other than Continuing or Ended

**Comment by bulletproof2k** [comment]
@evilhero I have done all the above steps and it does not solve the display problem. 💭

**Comment by rutskiuk** [comment]
Fixed for me, thanks

**Comment by bulletproof2k** [comment]
> Well it just gets curiouser, I can no longer sort by status, if I click to sort by status the continuing and ended are seemingly randomly scattered. Actually, not randomly at all, sorted by Title, alphabetically.
> I cannot see any removed, loading or failed series, I changed the view to show all and tried a search through both the browser and the filter, none show anything other than Continuing or Ended

How did you fix it?

**Comment by evilhero** [comment]
@bulletproof2k you're on master, the fix just went into python3-dev. You have to either switch to the dev branch or wait for the merge which is gonna be abit since we just did it.

I believe if you don't have your web browser width set to full/maximized it may not show the misalignment as well (on my machine whenever the browser window width was the maximum, it causes the misalignment).
